schema.graphql

```diff
@@ -800,6 +800,61 @@ type NablaSwapFee @entity {
     backstopPool: BackstopPool # link to the BackstopPool entity
 }
 
+type NablaSwap @entity {
+    id: ID! # calculated from blockNumber - extrinsicIndex
+    timestamp: DateTime! # need this to pull recent txns for specific token or pair
+    # from router Swap event
+    sender: String!
+    amountIn: BigInt!
+    amountOut: BigInt!
+    tokenIn: String!
+    tokenOut: String!
```

Comment:
Maybe we can hook these up to our existing [NablaToken](https://github.com/pendulum-chain/pendulum-squids/blob/fe71b51ae6a6dfa35e1c66a3b22ef73400d485cc/schema.graphql#L751) entity?

Reply:
Done